src/app/core/services/app-server/app-server.service.ts

```diff
@@ -180,7 +180,7 @@ export class AppServerService {
             subjectId
           ),
           updatedSubject,
-          { headers }
+          { headers, params: { forceFcmToken: 'true' } }
```

Comment:
is this version of the app server deployed to alpha?

Reply:
Not yet. Can I deploy it already?